tests/utils/test_walters.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import pandas
+
+from dlme_airflow.utils.walters import remove_walters_non_relevant
+from dlme_airflow.models.provider import Provider
+
+
+# This mock prevents writing to the fixture CSV when testing
+@pytest.fixture
+def mock_dataframe_to_csv(monkeypatch):
+    def mock_to_csv(_self, _filename):
+        return True
+
+    monkeypatch.setattr(pandas.DataFrame, "to_csv", mock_to_csv)
```

Comment:
@jacobthill I think instead of mocking the `to_csv` for the panda to simply return True, I think you probably want to instead mock the return value of line 47 of the code (`working_csv = coll.datafile("csv")`) to be a path to a fixture file and then the test will need to verify the changes to the file.

This will likely also need some setup/cleanup - like copy the fixture to a transient temp path and then cleanup afterwards so you don't overwriet the fixture. 

As-is, because the mock simply returns true from the function, the function isn't actually being tested.

Reply:
Got it, I'll push up a fix